cmd/bootkube/render.go

```diff
@@ -196,3 +217,28 @@ func altNamesFromURLs(urls []*url.URL) *tlsutil.AltNames {
 	}
 	return &an
 }
+
+func pickDNSServiceIP(serviceCIDR string) (string, error) {
+	ip, ipnet, err := net.ParseCIDR(serviceCIDR)
+	if err != nil {
+		return "", err
+	}
+	// pick the 10th IP as a conventional default
+	for i := 0; i < 10; i++ {
+		incIPv4(ip)
```

Comment:
Another option would be to have render actually output a simple cloud-config/ignition config - I don't think theres much that would have to be templated here (to at least provide an example). Then the hack examples could be updated to use that rendered config. - but this could be a follow-up issue.